src/codegen/ast_interpreter.cpp

```diff
@@ -630,6 +795,7 @@ Value ASTInterpreter::visit_invoke(AST_Invoke* node) {
         v = visit_stmt(node->stmt);
         next_block = node->normal_dest;
```

Comment:
Should we call finishJITing here?  ie am I reading this right -- in the invoke case, we will jit across basic blocks?  Which seems pretty reasonable but I'm not sure if it's intentional or not.